accepted/2020/better-obsoletion/better-obsoletion.md

```diff
@@ -235,6 +235,14 @@ follows:
   them in the IDE.
 * The compiler should assume that `UrlFormat` and `DiagnosticId` are independent
   features, in other words both can be used, either, or neither.
+* If `UrlFormat` is provided without a `DiagnosticId` the compiler will use the
+  `DiagnosticId` that it uses by default as the format argument to `UrlFormat`.
+  For C# this could be `CS0618`, `CS0619`, `CS1062`, or `CS1063`. The internal
+  error codes don't consider leading zeroes, but the padding is important when
+  we actually do the format substitution.
```

Comment:
```suggestion
  For C# this could be `CS0618`, for example, but will vary depending on context.
  The internal error codes don't consider leading zeroes, but
  the padding is important when we actually do the format substitution.
```

changing to just call out CS0618 as an example because the list I mentioned before was incomplete and we don't need to specify every last code here.